_i18n/de/resources/moneropedia/openalias.md

```diff
@@ -1,19 +1,19 @@
 ---
-terms: ["OpenAlias"]
-summary: "a standard that allows you to use an email or domain syntax to pay someone instead of an address, eg. donate@getmonero.org or donate.getmonero.org"
+terms: ["OpenAlias", "OpenAlias-Adresse"]
+summary: "Ein Standard, der es ermöglicht, zur Zahlung eine Mailadresse oder einen Domainsyntax (z.B. donate@getmonero.org oder donate.getmonero.org) anstelle einer komplizierten Adresse zu verwenden"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-The Monero Core Team released a standard called OpenAlias which permits much more human-readable addresses and "squares" the Zooko's triangle. OpenAlias can be used for any cryptocurrency and is already implemented in Monero, Bitcoin (in latest Electrum versions) and HyperStake.
+Das Monero-Core-Team hat mit der sogenannten OpenAlias einen Standard veröffentlicht, der Adressen mit besserer visueller Lesbarkeit erlaubt und Zookos Dreieck "quadriert". OpenAlias kann bei jeder Kryptowährung verwendet werden und ist bisher in Monero, Bitcoin (den letzten Electrum-Versionen) und HyperStake implementiert.
```

Comment:
```suggestion
Das Monero-Core-Team hat mit der sogenannten OpenAlias einen Standard veröffentlicht, der Adressen mit besserer visueller Lesbarkeit erlaubt und [Zookos Dreieck](https://en.wikipedia.org/wiki/Zooko%27s_triangle) "quadriert". OpenAlias kann bei jeder Kryptowährung verwendet werden und ist bisher in Monero, Bitcoin (den letzten Electrum-Versionen) und HyperStake implementiert.
```

I guess "squaring" Zooko's triangle could use an explanation... :) Since I didn't even know what Zooko's triangle is, I added a link to Wikipedia. Now at least I get what squaring it means :)

Reply:
Same for me. I find an added link here extremely useful, but once again I don't want to add something like that by myself. @erciccione

Reply:
> I don't want to add something like that by myself

You are correct, this is out of scope for this PR. Text changes should be added to the English version and then adapted to all other languages. It's very important to keep consistency between the files.